src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/prisonperson/PrisonPersonMigrationFilter.kt

```diff
@@ -11,4 +12,10 @@ class PrisonPersonMigrationFilter(
     example = "A1234BC",
   )
   val prisonerNumber: String?,
+
+  @Schema(
+    description = "The type of migration for this person, e.g. which data is being migrated",
+    example = "PHYSICAL_ATTRIBUTES",
+  )
+  val migrationType: PrisonPersonMigrationMappingRequest.MigrationType = PrisonPersonMigrationMappingRequest.MigrationType.PHYSICAL_ATTRIBUTES,
```

Comment:
Pass the migraiton type into the request - default to physical attributes so that migrations continue to work prior to changing the UI.